sormas-backend/src/main/java/de/symeda/sormas/backend/externaljournal/ExternalJournalService.java

```diff
@@ -137,10 +144,35 @@ private String getPatientDiaryAuthTokenInternal() {
 		}
 	}
 
+	/**
+	 * Notify external journals that a followUpUntilDate has been updated
+	 *
+	 * @param contact
+	 *            a contact assigned to a person already available in the external journal
+	 * 
+	 */
+	public void notifyExternalJournalFollowUpUntilUpdate(ContactDto contact) {
+		SymptomJournalStatus savedStatus = personFacade.getPersonByUuid(contact.getPerson().getUuid()).getSymptomJournalStatus();
+		if (SymptomJournalStatus.REGISTERED.equals(savedStatus) || SymptomJournalStatus.ACCEPTED.equals(savedStatus)) {
+			Date latestSavedFollowUpUntilDate = personFacade.getLatestFollowUpEndDateByUuid(contact.getPerson().getUuid());
+			if (contact.getFollowUpUntil().after(latestSavedFollowUpUntilDate)) {
```

Comment:
Is it possible that `getFollowUpUntil` returns `null` and we get a NPE?

Reply:
I think that is unlikely, because a contact automatically gets a FollowUpUntil date at creation. And even if it returns null, the error should already be logged.
Are my reasons convincing enough to leave it like that? If not, what would you like me to do?

Reply:
Yes, it is ok like this